docs/developer/workflows.md

```diff
@@ -66,3 +66,5 @@ These workflows will help with excluding certain terms from integration into Mon
 7. `reports/excluded_terms.txt`: Runs reports/%_term_exclusions.txt for all ontologies and combines into a single file.
 8. `reports/exclusion_reasons.robot.template.tsv`: Runs reports/%_exclusion_reasons.robot.template.tsv for all ontologies and combines into a single file. 
 9. `exclusions-all`: Runs all exclusion artefacts for all ontologies.
+10. `sync`: Runs synchronization pipeline.
```

Comment:
## Docs
FYI. Added the same to makefile `--help`

Reply:
I think Nico will have seen this by now, and also this kind of thing is expected for all `mondo-ingest` PRs; marking resolved.